src/components/Editor/EmptyLines.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import { Component } from "react";
+
+import actions from "../../actions";
+import { getSelectedSource } from "../../selectors";
+
+import getEmptyLines from "../../utils/parser/getEmptyLines";
+
+import "./EmptyLines.css";
+
+type props = {
+  selectedSource: SourceRecord,
+  editor: Object
+};
+
+class EmptyLines extends Component {
+  props: props;
+
+  disableEmptyLines: Function;
+
+  componentDidMount() {
+    this.disableEmptyLines();
+  }
+
+  componentDidUpdate() {
+    this.disableEmptyLines();
+  }
+
+  componentWillUnmount() {
+    const { selectedSource, editor } = this.props;
```

Comment:
Is this even worth doing or needed?  Since it requires another parsing/tokenizing of content, it could be an expensive cleanup.